apps/web/src/contexts/ABTestingContext/ABTestingContext.tsx

```diff
@@ -0,0 +1,40 @@
+import { FEATURE_FLAGS } from 'contexts/ABTestingContext/config'
+import { createContext, useCallback, useContext, useState } from 'react'
+
+interface ABTestingProviderProps {
+  children: React.ReactNode
+  ABTestUserResults: { [flag in FEATURE_FLAGS]: boolean }
+}
+
+type ABTestingContextType = {
+  featureFlagResults: { [flag in FEATURE_FLAGS]: boolean }
+  selectUserfeatureABResult: (flagKey: FEATURE_FLAGS) => boolean
+}
+
+const ABTestingContextManager = createContext({} as ABTestingContextType)
```

Comment:
do not use type assertion. If the initial value is empty let's just use `undefined` or `null`

Reply:
```suggestion
const ABTestingContextManager = createContext<ExperimentalFeatureContext | null>(null)
```